pkg/datastore/documentloader/defaults.go

```diff
@@ -67,17 +67,42 @@ func DefaultDocLoaderFunc(filetype string) func(ctx context.Context, reader io.R
 		return func(ctx context.Context, reader io.Reader) ([]vs.Document, error) {
 			return FromGolc(golcdocloaders.NewNotebook(reader)).Load(ctx)
 		}
-	case ".docx", ".odt", ".rtf", "application/vnd.oasis.opendocument.text", "text/rtf", "application/vnd.openxmlformats-officedocument.wordprocessingml.document":
+	case ".docx", ".odt", ".rtf", "text/rtf", "application/vnd.oasis.opendocument.text", "application/vnd.openxmlformats-officedocument.wordprocessingml.document":
```

Comment:
Do we actually support plain old `.doc`?

Reply:
I can test it out, I think it should be backward compatiable.

Reply:
It turns out that it could be pretty tricky to support doc. `.doc` is really a old format of doc and not many OSS libraries support it today. I think we can just say we don't support .doc and only newer format like `.docx` is supported. Even unidoc doesn't support it. https://help.unidoc.io/article/147-how-to-convert-ms-word-docx-file-into-pdf-using-unioffice